lib/tasks/stories.rake

```diff
@@ -0,0 +1,19 @@
+namespace :stories do
+  desc 'Import new stories from WRI RSS feed'
+  task import: :environment do
+    puts '##############################'
+    puts '# Starting to import stories #'
+    puts '##############################'
+    ImportStories.new.call
+    puts '########### ENDED ############'
+  end
+
+  desc 'Delete stories and re-import'
+  task fresh_import: :environment do
```

Comment:
I'd add a ref to this in the main `db:import` task

Reply:
That makes sense.